azure-pipelines/official.yml

```diff
@@ -1,17 +1,3 @@
-trigger:
-  batch: true
-  branches:
-    include:
-    - main
-    - microbuild
-    - 'validate/*'
-  paths:
-    exclude:
-    - doc/
-    - '*.md'
-    - .vscode/
-    - azure-pipelines/release.yml
-    - azure-pipelines/vs-insertion.yml
 #schedules:
```

Comment:
A pipeline YAML with no triggers specified automatically runs for PRs and maybe other things. This is why the `release.yml` file leads with this:

```yml
trigger: none # We only want to trigger manually or based on resources
pr: none
```

I believe you should add this to the top of this file as well.

Reply:
Done